gwsumm/data/utils.py

```diff
@@ -29,8 +29,8 @@
 from ligo.segments import segmentlist as LigoSegmentList
 
 from gwpy.segments import (DataQualityFlag, SegmentList, Segment)
-from gwpy.signal.fft import (get_default_fft_api, lal as fft_lal)
-from gwpy.signal.fft.registry import get_method as get_fft_method
+from gwpy.signal.spectral import (get_default_fft_api, _lal as fft_lal)
+from gwpy.signal.spectral._registry import get_method as get_fft_method
```

Comment:
@alurban, as we discussed offline, I think usage of `gwpy.signal.spectral` in this module can be removed, since as of gwpy-0.14.0, `get_default_fft_api` will always return `'scipy'`.

Reply:
@duncanmmacleod, awesome, this is done in the latest commit